tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -239,6 +240,7 @@ func (ls *leaseStresser) randomlyDropLeases() {
 			// if randomlyDropLease encountered an error such as context is cancelled, remove the lease from aliveLeases
 			// becasue we can't tell whether the lease is dropped or not.
 			if err != nil {
+				plog.Errorf("drop lease %v has failed error (%v)", leaseID, err)
```

Comment:
is this expected or unexpected? will we issue this rpc during failure injecting period?